src/components/Users/UserAvailabilityTab.tsx

```diff
@@ -170,6 +179,37 @@ export default function UserAvailabilityTab({ userData: user }: Props) {
                 setQParams={setQParams}
               />
             </Popover>
+          ) : (
+            <div
+              className={cn(
+                "grid h-full grid-rows-[1fr_auto_1fr] rounded-lg transition-all bg-gray-100",
+                "relative overflow-hidden opacity-50",
+              )}
+            >
+              <div />
+              <div className="flex flex-col items-center gap-2 relative z-20">
+                <span
+                  className={cn(
+                    "text-base",
+                    isToday ? "text-gray-900" : "text-gray-500",
+                  )}
+                >
+                  {date.getDate()}
+                </span>
+                <div className="flex justify-center gap-0.5">
+                  {templates
+                    ?.slice(0, 5)
+                    .map((template) => (
+                      <ColoredIndicator
+                        key={template.id}
+                        id={template.id}
+                        className="size-1.5 rounded-full"
+                      />
+                    ))}
+                </div>
+              </div>
+              <div />
+            </div>
```

Comment:
let's not duplicate code; we can simply disable the Popover's trigger instead if not schedulable resource.